cmd/restic/cmd_find_integration_test.go

```diff
@@ -91,3 +91,53 @@ func TestFindJSON(t *testing.T) {
 	rtest.Assert(t, len(matches[0].Matches) == 3, "expected 3 files to match (%v)", matches[0].Matches)
 	rtest.Assert(t, matches[0].Hits == 3, "expected hits to show 3 matches (%v)", datafile)
 }
+
+
+type testMatches2 struct {
+	Hits       int          `json:"hits,omitempty"`
+	SnapshotID string       `json:"snapshot,omitempty"`
+	Matches    []testMatch2 `json:"matches,omitempty"`
+}
+type testMatch2 struct {
+	Path        string    `json:"path,omitempty"`
+	Size        uint64    `json:"size,omitempty"`
+	Date        time.Time `json:"date,omitempty"`
+}
+
+func TestFindSorting(t *testing.T) {
+	env, cleanup := withTestEnvironment(t)
+	defer cleanup()
+
+	datafile := testSetupBackupData(t, env)
+	opts := BackupOptions{}
+
+	// first backup
+	testRunBackup(t, "", []string{env.testdata}, opts, env.gopts)
+	testRunCheck(t, env.gopts)
+
+	// second backup
+	testRunBackup(t, "", []string{env.testdata}, opts, env.gopts)
+
+	// first restic find
+	results := testRunFind(t, true, FindOptions{}, env.gopts, "testfile")
+	lines := strings.Split(string(results), "\n")
+	rtest.Assert(t, len(lines) == 2, "expected two files found in repo (%v), found %d", datafile, len(lines))
```

Comment:
Please also check that find returns the results in the correct order. (The simplest way is probably to call `testListSnapshots(...)` after creating the first snapshot to get the first ID)